includes/Modules/Analytics.php

```diff
@@ -562,6 +563,22 @@ protected function create_data_request( $method, $datapoint, array $data = array
 						}
 						return false;
 					};
+				case 'tag-permission':
+					return function() use ( $data ) {
+						if ( ! isset( $data['tag'] ) ) {
+							/* translators: %s: Missing parameter name */
+							return new WP_Error( 'missing_required_param', sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'tag' ), array( 'status' => 400 ) );
```

Comment:
Please split this onto multiple lines. The `sprintf` can be a single line as it only has a single parameter. If it were longer then that should be split up as well. Thank you!